benchmark/ngu_formula.py

```diff
@@ -0,0 +1,52 @@
+"""
+NGU alpha formula.
+
+Idea:
+    The more the instruments fell, the more we buy.
+"""
+
+from thousandaire.alpha import BaseAlphaFormula
+from thousandaire.data_classes import Portfolio
+
+def make_list(multi, days):
+    """
+    Make position list.
```

Comment:
Can you explain a little bit more about what is position and how is this used?

Reply:
Position means how much money we will put on the investing target.
This idea came from a gambling strategy:
I bet ＄1 on the first game, but I lost.
Then, I would bet ＄2 on the second game.
If I lose again, I would raise to ＄4 on the third game, and so on.
In this strategy, I could win money when I win only one game.

Reply:
In the thinking, I build this alpha,  multi and days come from parameters in ngu_settings.py
multi means how many times of bets I would put in the next run.
If it is two, my bet list would be  [0, 1, 2, 4, 8, 16 ...]
If it is three, it would be [0, 1, 3, 9, 18 ...] 
and 'days' means how many runs I will raise my bets. If multi is 2 and days is 10 , the I would bet up to 1024.

Reply:
And this list is to keep how many positions I put on a single instrument in the last run.
If the price falls today, it means I lose that I should buy more, so the index would be plus one  to a higher position.

Reply:
i see, thanks!